File: phases/00-setup-and-tooling/04-apis-and-keys/code/test_api_call.ts
import OpenAI from "openai";
import "dotenv/config";

const client = new OpenAI({
  apiKey: process.env.DEEPSEEK_API_KEY,
  baseURL: "https://api.deepseek.com",
});

const MODEL = process.env.LLM_MODEL ?? "deepseek-chat";

const response = await client.chat.completions.create({
  model: MODEL,
  max_tokens: 256,
  messages: [{ role: "user", content: "Что такое нейросеть в одном предложении?" }],
});

const content = response.choices[0].message.content;

process.stdout.write((response.choices[0].message.content ?? "") + "\n");

File: phases/00-setup-and-tooling/04-apis-and-keys/code/test_api_call.py
from dotenv import load_dotenv
import os
from openai import OpenAI

load_dotenv()
client = OpenAI(
    api_key=os.getenv("DEEPSEEK_API_KEY"),
    base_url="https://api.deepseek.com"
)

MODEL = os.environ.get("LLM_MODEL", "deepseek-chat")

response = client.chat.completions.create(
    model=MODEL,
    max_tokens=256,
    messages=[{"role": "user", "content": "Что такое нейросеть в одном предложении?"}]
)

print(response.choices[0].message.content)
